evaluations.html

```diff
@@ -2,49 +2,51 @@
 <html>
 	<head>
 		<title>Evaluations</title>
-		<link rel="stylesheet" type="text/css" href="evaluations.css">
+		<link rel="stylesheet/css" type="text/css" href="assets/css/normalize.css" />
+        <link rel="stylesheet/less" type="text/css" href="assets/css/evaluations.less">
```

Comment:
normalize css poate fi inclus in evaluations less si tot css se incarca atunci cu un singur import